### PR TITLE
Feature/add booking helper #000

### PR DESCRIPTION
**Pull request by mrturkmencom**
- calculating cost of events handled in store side rather than haaukins daemon. 
- gRPC call regarding to getting timeSeries added 
- some functions are updated according to need

### CODE REVIEW

File: database/timeline.go [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+		err := r.Scan(&event.Id, &event.Tag, &event.Name, &event.Available, &event.Capacity, &event.Status, &event.Frontends,
+			&event.Exercises, &event.StartedAt, &event.ExpectedFinishTime, &event.FinishedAt)
+		if err != nil && !strings.Contains(err.Error(), "Null conversion error ") {
+			log.Fatalf("Error on scanning query %v", err)
+		}
+		events = append(events, *event)
+	}
+
+	return events
+}
```
Comment (gianmarcomennecozzi) [inline database/timeline.go:83]: The function `func (s *store) GetEvents(in *pb.GetEventRequest) ([]model.Event, error)` we have in the `store.go` can be used. No need to create a new function. i think we can pass `nil` as argument

Comment (mrturkmencom) [reply database/timeline.go:83]: That cannot be used.

**Comment by mrturkmencom** [comment]
>     * Too many logs in the functions `GetEventStatus`, `SetEventStatus`.
> 
>     * Closing event function dosent work well (it dosent return any error) but from the store you can see (SS) that it creates another row in the DB and moreover the `finished_at` time didn't change
>       Screenshot from 2020-07-04 14-15-22
> 
> 
> I can not understand the meaning of changing the event tag of the event......
> 
> i like the timeline feature and its working well but apart from that i don't like the modification you made.

You did not understand it,  maybe the reason of  that is, you did NOT pay attention what I wrote on slack, issues, pull requests, on code etc.

**Comment by gianmarcomennecozzi** [comment]
Hmm yeah maybe i didn't understand what you said in slack. But if you want to keep the function that change the event tag there are some stuff to change anyway cause it's working but not as expected i think

**Pull request merged**